### DISCUSSION TITLE
How to read the source code ?

### DISCUSSION THREAD

**Discussion by sathishsoundharajan** [Netflix/conductor#2827] [General]
Hello Everyone, Do we have guide / document on where to start OR how to read the source code ?

**Comment by apanicker-nflx** [comment]
Conductor is built on the SpringBoot framework. Each module represents a pluggable component within the system. The `core` module houses the orchestration engine, state machine and the core configurations for running the server. The persistence layer, indexing layer, lock implementation, external payload storage are all components that can be plugged in as per your requirement.
